hedera-node/docs/design/services/smart-contract-service/accounts-blocklisting.md

```diff
@@ -0,0 +1,73 @@
+# Accounts Blocklisting
+
+## Purpose
+
+In Ethereum, everyone that knows a private key is able to authorize a transaction from the corresponding EVM public address, and the first to send a transaction would be eligible to withdraw any funds from that address.
+In Hedera, it is possible for an account with associated EVM address alias to have a key that does not correspond to the alias. When an account is created with EVM address alias matching some publicly known exposed private key, this is what we refer to as “EVM address squatting”.
```

Comment:
Should we use "reserve" instead of "squat"?
Squat may have a negative connotation.

Reply:
Replaced all occurrences of "squat" with "reserve".